common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -238,6 +239,8 @@ object LoggingMarkers {
   // Check invoker healthy state from loadbalancer
   val LOADBALANCER_INVOKER_OFFLINE = LogMarkerToken(loadbalancer, "invokerOffline", count)
   val LOADBALANCER_INVOKER_UNHEALTHY = LogMarkerToken(loadbalancer, "invokerUnhealthy", count)
+  def LOADBALANCER_ACTIVATION_START(namespaceId: String) =
```

Comment:
should this take a string type than string? entity name or identity perhaps.

Reply:
The alternative type here would be `EntityName` and I don't see much value in adding those.